ompi/mca/pml/ob1/pml_ob1_recvreq.c

```diff
@@ -92,7 +90,6 @@ static int mca_pml_ob1_recv_request_free(struct ompi_request_t** request)
         MCA_PML_OB1_RECV_REQUEST_RETURN( recvreq );
     }
 
```

Comment:
Because there is no locking anymore. Everything related to the synchronization is now handled down in the sync_wait, via atomic operations.